wasm-init/src/wasm.rs

```diff
@@ -50,12 +49,15 @@ pub fn wasm_init() {
 #[macro_export]
 macro_rules! __wasm_init_impl {
 	($gensym:ident, $($input:tt)*) => {
-		$crate::__macrodeps::item! {
-			#[$crate::__macrodeps::wasm_bindgen]
-			pub fn [<__wasm_init $gensym>]() {
+		const _: () = {
+			#[export_name = $crate::__macrodeps::concat!(
+				"__wasm_init",
+				$crate::__macrodeps::stringify!($gensym)
+			)]
+			pub unsafe extern "C" fn init() {
```

Comment:
Does this need to be unsafe?

Reply:
I honestly have no idea. I assume no, but all exports are generated that way from wasm-bindgen? The only reason it was there was because I started with what wasm-bindgen was generating. Nobody is even able to touch the function without going through wasm lookup from javascript so.. the `unsafe` has no effect anyway so I'll remove it, it just serves to confuse.